CHANGELOG.md

```diff
@@ -6,6 +6,11 @@
 
 - Client factories for Buzz.
 
+### Changed
+
+- **[BC] PluginClientFactory returns an instance of `Http\Client\Common\PluginClient`** (see [php-http/client-common#14](https://github.com/php-http/client-common/pull/14))
+- Plugins are loaded from their new packages, fall back to `php-http/plugins` if necessary
+
```

Comment:
I am fine with that as well. In that case, we don't need to rely on the old package.